src/test/java/executor/service/services/step/StepExecutionClickXpathTest.java

```diff
@@ -0,0 +1,121 @@
+package executor.service.services.step;
+
+import executor.service.model.Step;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledIf;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.io.File;
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StepExecutionClickXpathTest {
+
+    private final String headerXpath = "//*[@id=\"header-blocks--2\"]/div[1]/div/div[3]/div/div/h3/span[1]";
+    private final String wrongHeaderXpath = "//*[@id=\"header-blocks--2\"]/div[1]/div/div[3]/div/div/h3/span[2]";
+    private final File file = new File("src/main/resources/chromedriver.exe");
+    private StepExecutionClickXpath stepExecutionClickXpath;
+    private Step step;
+    private Step notInitializedStep;
+    private WebDriver webDriver;
+    private WebDriver notInitializedWebDriver;
+
+    static {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+    }
+
+    @BeforeEach
+    void setUp() {
```

Comment:
let's use common code style
```suggestion
    public void setUp() {
```

Reply:
this is not the way to solve this problem

Reply:
Done